### PR TITLE
Fix comment

### PR DESCRIPTION
**Pull request by Sumanai**

### CODE REVIEW

File: controller/main_controller.php [hunk 1]
```diff
@@ -93,7 +93,7 @@ public function display($route)
 	* @param string $route The route name for a page
 	* @return object $entity The entity object
 	* @throws http_exception
-	* @access public
+	* @access protected
```
Comment (Nicofuma) [inline controller/main_controller.php]: @VSEphpbb Honestly you shouldn't have to specify this annotation. It's coming from php4 ;)

Comment (Sumanai) [inline controller/main_controller.php]: Well then need to delete the @ access annotation in all files (including those in the core of phpBB).

Comment (Nicofuma) [inline controller/main_controller.php]: In the core it is still used sometimes yes (mostly when the visibility isn't specified) but it should be changed yes.

Comment (VSEphpbb) [inline controller/main_controller.php]: @Nicofuma is there any reason not to have it... as in, is it worth all the work to remove it from everywhere?

Comment (Nicofuma) [inline controller/main_controller.php]: well... it is useless

Comment (Sumanai) [inline controller/main_controller.php]: I finally made friends TortoiseHg and Git (sorry for the couple commits trash, checking the work), so I can do normal commit.
I can remove these lines in the project, if no one wants to do it.

**Pull request closed without merging**